ouroboros-consensus/src/Ouroboros/Consensus/Storage/ChainDB/API.hs

```diff
@@ -171,6 +171,19 @@ data ChainDB m blk = ChainDB {
       -- | Return the LedgerDB containing the last @k@ ledger states.
     , getLedgerDB        :: STM m (LedgerDB' blk)
 
+      -- | Acquire a value handle and ledger DB truncated to the specified point
```

Comment:
What is the relationship between the two values returned? Any chance we can write this relationship in the form of a postcondition?

Reply:
Added a clarifying comment